variables.tf

```diff
@@ -77,10 +77,9 @@ variable "ec2_ingress_sg_id" {
 }
 
 //----------------------------------------------------------------------
-// ECS Cluster Variables
+// ECS Cluster Variables -- And setup a toggle (true/false)
 //----------------------------------------------------------------------
 variable "container_insights_enablement" {
   description = "Whether container sights are set, valid values are [enabled,disabled]"
-  type        = string
-  default     = "disabled"
+  container_insights_enablement     = 1
```

Comment:
This is an invalid `variable` declaration. The argument `container_insights_enablement` has no place here.

You've also removed the `type` and `default` arguments.